src/main/scala/jtag/Utils.scala

```diff
@@ -12,40 +13,23 @@ class Tristate extends Bundle {
   val driven = Bool()  // active high, pin is hi-Z when driven is low
 }
 
-class NegativeEdgeLatch[T <: Data](clock: Clock, dataType: T)
-    extends Module(override_clock=Some(clock)) {
-  class IoClass extends Bundle {
-    val next = Input(dataType)
-    val enable = Input(Bool())
-    val output = Output(dataType)
-  }
-  val io = IO(new IoClass)
-
-  val reg = Reg(dataType)
-  when (io.enable) {
-    reg := io.next
-  }
-  io.output := reg
-}
-
 /** Generates a register that updates on the falling edge of the input clock signal.
   */
 object NegativeEdgeLatch {
```

Comment:
This isn't germane to your pull request, but shouldn't this be called NegativeEdgeReg (or perhaps NegEdgeReg)?  It's not level-sensitive, so Latch confused me.